src/components/ThoughtForm.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+// eslint-disable-next-line react/prop-types
+export const ThoughtForm = ({ onFormSubmit }) => {
+  // Declare a state variable to hold the user's input
+  // 'message' is the current value of the input, and 'setMessage' is used to update it
+  const [message, setMessage] = useState("");
+  // Function to handle form submission
+  const handleSubmit = (event) => {
+    // Prevent the form's default behavior of reloading the page when submitted
+    event.preventDefault();
+    if (message.length < 5 || message.length > 140) {
+      alert("Message must be between 5 and 140 characters."); // Show an alert if it's not valid
```

Comment:
Great work with the validation! One improvement could be to replace the alert with a more modern approach, like an inline error text, which would give a more seamless UX.